CONTRIBUTING.md

```diff
@@ -29,11 +29,20 @@ If a maintainer asks you to "rebase" your PR, they're saying that a lot of code
 $ git clone https://github.com/keenlabs/keen-sdk-net.git && cd keen-sdk-net
 ```
 
-Now, launch Visual Studio and open `Keen.sln`. Build the solution, discover unit tests, and run them. Set some breakpoints and get acquainted with the PCL, the .NET 4.5 DLL and the .NET 3.5 DLL, as well as the related test projects. Maybe add a mini test project that references one of the built binaries and play with actually using the SDK as per the instructions in the [README](./README.md).
+Now, launch Visual Studio and open `Keen.sln`. Build the solution, discover unit tests, and run them. Or, if using the .NET Core SDK tools, build and make sure tests pass:
+
+```ssh
+$ dotnet build
```

Comment:
Does this actually work these days from the root of the project?

Reply:
Yeah it will, but I should probably mention `error MSB3644` or explicitly passing `-f netstandard2.0` on platforms where you don't/can't have the right SDK/targeting pack for netfx4.5.

Reply:
Fun fact: running `dotnet test` from the root of the solution works too, which I didn't realize.